strongbox-cron/strongbox-cron-rest/src/test/resources/etc/conf/strongbox-cron-tasks.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cron-tasks-configuration>
+
+    <cron-task-configuration>
+        <name>ClearRepositoryTrashCronTask</name>
+        <one-time-execution>false</one-time-execution>
+        <immediate-execution>false</immediate-execution>
+        <properties>
+            <cronExpression>0 11 11 11 11 ? 2100</cronExpression> <!-- mandatory -->
+            <jobClass>org.carlspring.strongbox.cron.jobs.ClearRepositoryTrashCronJob</jobClass> <!-- mandatory -->
+        </properties>
+    </cron-task-configuration>
+
+    <cron-task-configuration>
+        <name>ClearRepositoryTrashCronJob</name>
+        <one-time-execution>false</one-time-execution>
+        <immediate-execution>false</immediate-execution>
+        <properties>
+            <cronExpression>0 11 11 11 11 ? 2100</cronExpression> <!-- mandatory -->
+            <jobClass>org.carlspring.strongbox.cron.jobs.ClearRepositoryTrashCronJob</jobClass> <!-- mandatory -->
+            <storageId>storage-common-proxies</storageId> <!-- optional -->
+            <repositoryId>maven-central</repositoryId> <!-- optional -->
+        </properties>
+    </cron-task-configuration>
+
+    <cron-task-configuration>
+        <name>RegenerateChecksumCronJob</name>
+        <one-time-execution>false</one-time-execution>
+        <immediate-execution>false</immediate-execution>
+        <properties>
+            <cronExpression>0 11 11 11 11 ? 2100</cronExpression> <!-- mandatory -->
+            <jobClass>org.carlspring.strongbox.cron.jobs.RegenerateChecksumCronJob</jobClass> <!-- mandatory -->
+            <storageId>storage-common-proxies</storageId> <!-- optional -->
+            <repositoryId>maven-central</repositoryId> <!-- optional -->
+            <basePath>org</basePath> <!-- optional -->
+            <forceRegeneration>true</forceRegeneration> <!-- optional, default false -->
+        </properties>
+    </cron-task-configuration>
+
+    <cron-task-configuration>
+        <name>DownloadRemoteMavenIndexCronJob</name>
```

Comment:
I think we need some sort of format for these... Especially, if they're system generated ones. Perhaps something like "Download remote Maven Indexes for ${storageId}:${repositoryId}".

I mean... this XML file will be the one going into production. :)